Create ui-c8y-1021-22-11-alias-cannot-be-empty-7817-graft-release-y-2025-7883.md (#2791)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-22-11-alias-cannot-be-empty-7817-graft-release-y-2025-7883.md

---
date: ""

> title: alias cannot be empty (#7817) [GRAFT][release/y2025] (#7883)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Login alias can no longer be an empty string
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61921
version: 1021.22.11
---

> alias cannot be empty (#7817) [GRAFT][release/y2025] (#7883)

Author: The Claude AI suggested to use the following body:
```suggestion
In some cases, it was possible to create an empty alias for a device, which could lead to unexpected behavior and errors in the system. This issue has now been fixed. Devices can no longer be created with an empty alias. Existing devices with an empty alias will continue to work, but it is recommended to update their aliases to a non-empty value.
```

> In some cases, it was possible to create an empty alias for a device, which could lead to unexpected behavior and errors in the system. This issue has now been fixed. Devices can no longer be created with an empty alias. Existing devices with an empty alias will continue to work, but it is recommended to update their aliases to a non-empty value.

Reviewer: ```suggestion
The login alias can no longer be an empty string, that is, a string consisting only of whitespace, null, or undefined. If no login alias is provided for the user, this field must not be included in the user object.
```
